contrib/mesos/docs/ha.md

```diff
@@ -0,0 +1,66 @@
+## High Availability
+
+### Scheduler
+
+Implementation of scheduler HA features has started and includes:
+
+- Checkpointing by default (`--checkpoint`)
+- Large failover-timeout by default (`--failover_timeout`)
+- Hot-failover w/ multiple scheduler instances (`--ha`)
+- Best effort task reconciliation on failover
+
+#### Multiple Instances
+
+Multiple scheduler instances may be run to support a hot-failover scenario in which one scheduler fails and another takes over immediately.
```

Comment:
I don't think this is actually a hot-standby scenario, since the standby schedulers don't continuously track the master state